app/components/menu_item/__snapshots__/index.test.tsx.snap

```diff
@@ -38,13 +38,16 @@ exports[`DrawerItem should match snapshot 1`] = `
     >
       <View
         style={
-          Object {
-            "alignItems": "center",
-            "height": 50,
-            "justifyContent": "center",
-            "marginLeft": 5,
-            "width": 45,
-          }
+          Array [
+            Object {
+              "alignItems": "center",
+              "height": 50,
+              "justifyContent": "center",
+              "marginLeft": 5,
+              "width": 45,
+            },
+            undefined,
```

Comment:
This refers to iconContainerStyle which is indeed undefined in the test.

Reply:
I know. I am just considering if we should be passing it if it is undefined. Seems like we are appending undefined to a list, instead of not appending anything if it is undefined. Not a big deal either way.